src/DynamoRevitIcons/RevitNodesImages.resx

```diff
@@ -9057,75 +9057,122 @@
+        SVeDaSSP5TgvjE76hTQvUD9z+udJM4qCHhqfS9vMXpRha8BVdciWqvQGKc4BJsUqKP/d3SE8OqNjJmR3
+        PI/2/qHUtrjjYEDe5JlT1l5K6A1yzpqLFKcCc1iSiIbltBLnZnQOTNKLCRpHP2f86s6JKuhxu7Izo9W8
+        QNgb7LqJxZmSFKcCc2Iv8MPKUCbk01hbO8TVM0sa9m9SFPTQeG5PdkYwc1GGrQXXJ3ZtsHjdIMUpwBg1
+        F5ip/q72PcP3nO7uxpElDMusJq+e0fjelqcyay3CAk6u7OcgxSnAmDh+d70AR2+gz5BUJjS2eo4pryro
+        5dI2i27KsLUIE3Tu6YcgxSkIk3iXD2ZCTvdDlpwz0mtfpNeMotuj8VW2PA8h7D0s2hKV4j5gSByKGJCh
+        lJN3+1gmFBd1vnd4ikwEja++5TmVkCUtmhekuA8YEmtEX4LC6PTefZlQr0iHf3efg4D+7N5j9Lo9Gr/a
+        luc+OPy5a+eb7hCkuA+YwT0Bb863PB6QCfVST/y7FzAQV+LF4ggGrLblOQbXJnZdsGhyluI+gjGe2ZmQ
+        e39RWwoaUavn1bc8h7Drkvi95iDFMWDExWCM0cuEMJz8mXSVCfW6L/49VFvqCnpnJ9ftNdIr6MGAvOXJ
+        0jZ7Q6tMya5L/HeaixTHgAl5UsawYcbMyYTinDGUUflAenqZCAzobXl6mEZyIcfFFSfS2gHy1/LfaS5S
+        HAMmdEFwAfjdvTY7E1K1JeouMD+ko6FWz3nLcyo1AuQ/L36nOUhxDJgQM6NP3WsqEyomZ/d+C1heJ/Do
+        dB7fTeee3JMITODqmeVtzg/ca8iT9SFMDZD/G/995iLFMWBADIJt3qtMqJic3ecUAUt6lxElnfMPg2Kf
+        YUwqmsEcBsd+01A9QF6P156DFMeAAUUQwJJMyGdUvnBnPcmGMs+iFSqBcTUD1DZFhQExCJ8NZEIWgLFM
+        yGdUKpDxWiTPNWsBU1WAxuadtos1GNArWWDY+C+ddxMmjnJyTq8NZkJJI92uHI6xkkoY2CIAMMFMWzU4
+        BNfgKp3XYimdQWKAFu2wSXEMmNALwlcfvPcLjhzHOwNwnJIJ+dT1oguMBdIPZUbv8wwYkNcJDhrDTR+a
+        xPpSkwAdihTHgBHFYs29VkzO7rWid/AY9YQq4PUmdgJjeVfmbc6JnLsASXEfwiAGZnEmlDQLiqd4CIzA
+        vNnrgz0cJUBS3AdMiQW8+8CGj5gJyd6B86mZ0NBvFfJeArHiHZ8PYk7PfWfm+Ewn9/xWbSoMeDf+A84H
+        DFCVHxxKcR8wJu4B/5uOMROSvUPpQTPy0OVB4/M+AplauFMBIv6zDqQYKucgxX3AGJW59AzD+ZxMKG4S
+        EfYs9fQdM6E8D9BEZfhcuBfBAPFJbwaIO2czAtSb/+YixSkkk7xpH7nXhop4UzMhUjQMjWWxLs8DXL22
+        2MhRAbLvkFg0LElxCjApjt+d2TjKdYLSg+aR3RuNzc8aEVXPaUHNDR0ixSnAKDWEMEuakwnFnTPSm9gN
+        NDY/Z0RaPvAV4bXdd5GJwxykOBUYFvP57935vkwoTu5ELv/RUGYiueEtHwJWhGxrcR1LilOBaSqjIYdk
+        Qrmc7UEjOQ/kiZi7Zy039CPiEfnFaaoU55DM82b+5l7zmZCVvL1myJIEQSOZm3cN5kTc8uFfBSdm+z6g
+        2PM+BCnOAeapdJULNJ8JqeqqR+4PoJF5E5+s/UPBKYT/DMPioYhIcS4wMa6gyY/puC8TkhVINLBXmOMm
+        ijKlJbwJ3Hc6Pz8SITCSWVFxh6Mn/I2jpaiqJDGUCfUWZK2frlNwHgq9YFH52iPFQ4Chvck2PabyF2CA
+        Jm/OoHHFgqzVjwPHCHNBtV5ApHgoMFWlnWoOoCYbgcbJwtwxCYszUq0XECkuAeaqBVhkKBPqFeZ49ylT
+        WsJhKKwLqvxEyiPFJcDgoQnYGCpJcB7Ija1VmFtKfKoCLCrWKaS4FBg9FIihzRnOA/lJh2MvyAyWyO07
+        JaoOQ4YUawHTfUY0WGNB43qFuWMvyEhIR8niGtEQUqwJzOczRjITImgcd6pyY49ZmDNED2C2Vi0bikix
+        FWhYrzB37AUZh0AxB6waACLFVqBxeUFGWv++wMMhUOxFrx4AIsVWoIFMSXuBoBEtN2t494eFmMF5avUA
+        ECm2BA1lappXyAaHhbVXypx8QynCWCULGkKKrUGjmaL2KqYGN3BqrpoZWN75A+YzTa6+DtiHFI8FDQB5
+        /8BD09g7GJC5awiO98y6Rp6e4JDY9O73SPHYwBCWsWUwDAaFpnIy550dmfjISmc+aDL2DyHF8wLMYc9g
+        Qa83eVeAc9D2/0+YCwxj7+C8UUziE2AQme3Q+CqPLtZEim8DMJO9hIHhcKKo+rzomkhxoy1S3GiLFDfa
+        IsWNtkhxoy1S3GiLFDfaIsWNtkhxoy1S3GiLFDfaIsWNluze+R+dPFk+KUnfeAAAAABJRU5ErkJggg==
+</value>
+  </data>
+  <data name="Revit.Elements.Element.GetSuperComponent.Large" type="System.Drawing.Bitmap, System.Drawing" mimetype="application/x-microsoft.net.object.bytearray.base64">
```

Comment:
we changed the node name to `GetParentComponents`, the files in resources should update to match